frontend/src/components/question-creation/QuestionSidebarItem.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import { Box, Icon, Text, VStack, WrapItem } from "@chakra-ui/react";
+import { useDrag } from "react-dnd";
+
+import { QuestionElement } from "../../types/QuestionTypes";
+import { DragTypes } from "../../types/DragTypes";
+
+interface QuestionSidebarItemProps {
+  element: QuestionElement;
+  addItem: (newQuestionElement: QuestionElement) => void;
+  icon: () => React.ReactElement;
+}
+
+interface DropResult {
+  name: string;
+}
+
+const QuestionSidebarItem = ({
+  element,
+  addItem,
+  icon,
+}: QuestionSidebarItemProps): React.ReactElement => {
+  const [{ isDragging }, drag] = useDrag(() => ({
+    type: DragTypes.QUESTION_ELEMENT,
+    item: { element },
+    end: (item, monitor) => {
+      const dropResult = monitor.getDropResult<DropResult>();
+      if (item && dropResult) {
+        addItem(element);
+      }
+    },
+    collect: (monitor) => ({
+      isDragging: monitor.isDragging(),
+      handlerId: monitor.getHandlerId(),
+    }),
+  }));
+
+  const opacity = isDragging ? 0.4 : 1;
+  return (
+    <Box ref={drag} style={{ opacity }}>
+      <WrapItem cursor="grab">
```

Comment:
Currently, the cursor changes appropriately when hovering over the `QuestionSidebarItem`. However, I can't get the cursor to change while dragging. According to the [linked thread](https://github.com/react-dnd/react-dnd/issues/325), this is how the HTML API for drag and drop works and the fix is to implement a custom API. Given this, leaving the cursor as is.